app/Models/Multiplayer/Room.php

```diff
@@ -688,7 +692,8 @@ private function assertValidStartPlay(User $user, PlaylistItem $playlistItem)
         }
 
         if ($playlistItem->max_attempts !== null) {
-            $playlistAttempts = $playlistItem->scoreLinks()->where('user_id', $user->getKey())->count();
+            $playlistAttempts = $playlistItem->scoreTokens()->where('user_id', $user->getKey())->whereNull('score_id')->count()
```

Comment:
What happens when score tokens finally get purged? Or will there be something that keeps score tokens for playlist items that are still playable? @peppy

Reply:
I'm not yet sure. Arguably we could keep score tokens around long enough that this didn't matter, but that could be months worth..

Shall we open a separate issue to track this? Might need a new table storage if we want this functionality.

Reply:
I'll add the count to scores_high table later similar to current rooms_high (and make sure it doesn't get reset etc)